polymer/src/amp-my-embed.html

```diff
@@ -0,0 +1,285 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../node_modules/@npm-polymer/polymer/polymer-element.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-flex-layout/iron-flex-layout.html">
+<script async src="https://cdn.ampproject.org/shadow-v0.js"></script>
+
+<dom-module id="amp-my-embed">
+
+  <template>
+
+    <style>
+
+      :host {
+        @apply --layout-vertical;
+        @apply --layout-center-justified;
+        text-align: center;
+        margin: 0 48px;
+      }
+
+      .title {
+        color: var(--app-primary-color);
+        font-weight: bold;
+      }
+
+      .description {
+        color: var(--app-secondary-color);
+      }
+
+      @media (max-width: 767px) {
+        :host {
+          margin: 0 12px;
+        }
+      }
+
+    </style>
+
+    <span id="name"></span>
+
+  </template>
+
+  <script>
+
+    class AmpMyEmbed extends Polymer.Element {
+
+      static get is() { return 'amp-my-embed'; }
+
+      static get properties() { return {
+
+        url: {
+          type: String,
+          value: window.location.origin
+        },
+        fullUrl: {
+          type: String,
+          computed: 'computeFullUrl(itemUrl)',
+          observer: 'urlChanged'
+        },
+        itemUrl: {
+          type: String,
+          notify: true
+        },
+        content: {
+          type: String,
+          notify: true
+        }
+      };
+    }
+
+      computeFullUrl(itemUrl) {
+        return [this.url, this.itemUrl, "embed"].join('')
```

Comment:
You are implicitly relying on these values. It is better to have the `computed` on the property as `_computeFullUrl(url, itemUrl)` such that you are always sure that `url` is properly set to compute the full url.

Reply:
Changed, thanks